### PR TITLE
update prestashoperpconnect with the latest version of the connector

### PR DESCRIPTION
**Pull request by arthru**

### CODE REVIEW

File: prestashoperpconnect/unit/import_synchronizer.py [hunk 1]
```diff
@@ -134,11 +134,8 @@ def run(self, prestashop_id):
         self._import_dependencies()
 
         erp_id = self._get_openerp_id()
-        self.mapper.convert(self.prestashop_record)
-        if erp_id:
-            record = self.mapper.data
-        else:
-            record = self.mapper.data_for_create
+        map_record = self.mapper.map_record(self.prestashop_record)
+        record = map_record.values()
```
Comment (guewen) [inline prestashoperpconnect/unit/import_synchronizer.py]: With you change, you always take all the values, you lost the 'for create' option.

```
if erp_id:
    record = map_record.values()
else:
    record = map_record.values(for_create=True)
```

ref: http://openerp-connector.com/api/api_mapper.html#connector.unit.mapper.MapRecord.values

File: prestashoperpconnect/unit/import_synchronizer.py [hunk 2]
```diff
@@ -674,15 +672,12 @@ def run(self, prestashop_id):
...
+        mapped = self.mapper.map_record(prestashop_record)
 
         if erp_id is None:
             erp_id = self._get_openerp_id()
 
-        if erp_id:
-            record = self.mapper.data
-        else:
-            record = self.mapper.data_for_create
+        record = mapped.values()
```
Comment (guewen) [inline prestashoperpconnect/unit/import_synchronizer.py]: Same comment as above

File: prestashoperpconnect/unit/import_synchronizer.py [hunk 3]
```diff
@@ -546,12 +544,12 @@ def _import_dependencies(self):
...
-            self._check_refunds(record['id_customer'], record['id'])
-        if record['id_carrier'] != '0':
-            self._check_dependency(record['id_carrier'],
-                                   'prestashop.delivery.carrier')
+        #if not('so_refund_no_dep' in self.session.context
+        #       and self.session.context['so_refund_no_dep']):
+        #    self._check_refunds(record['id_customer'], record['id'])
+        #if record['id_carrier'] != '0':
+        #    self._check_dependency(record['id_carrier'],
+        #                           'prestashop.delivery.carrier')
```
Comment (guewen) [inline prestashoperpconnect/unit/import_synchronizer.py]: Why is it commented? Can you remove the lines?

Comment (arthru) [inline prestashoperpconnect/unit/import_synchronizer.py]: This is going to be moved to another module as it depends on a plugin on prestashop...

**Comment by arthru** [comment]
Thanks for your review, I've fixed things from your comments...

**Comment by guewen** [comment]
Thanks!
:+1:

File: prestashoperpconnect/unit/import_synchronizer.py (hunk 1 above)
Comment (waltervargas) [inline prestashoperpconnect/unit/import_synchronizer.py]: Thanks @guewen , I lost too the 'for create' option.  [1] 

https://github.com/waltervargas/prestashoperpconnect/commit/73c510727eb2c621dedc5bf0279e9a66b08f2ccc

**Comment by sebastienbeau** [comment]
Thanks arthru for this update ! :+1:

**Pull request merged**